lucille-core/src/main/java/com/kmwllc/lucille/util/ElasticsearchUtils.java

```diff
@@ -23,58 +20,6 @@
  */
 public class ElasticsearchUtils {
 
```

Comment:
Removed getElasticsearchRestClient as RestHighLevelClient is deprecated and method is not being used anywhere else in Lucille.